docs/lease-metrics.md

```diff
@@ -0,0 +1,8 @@
+# Leases Metrics
+
+| Metric name| Metric type | Labels/tags | Status |
+| ---------- | ----------- | ----------- | ----------- |
+| kube_lease_created | Gauge | `lease`=&lt;lease-name&gt; | EXPERIMENTAL |
+| kube_lease_labels | Gauge | `lease`=&lt;lease-name&gt; <br> `label_LEASE_LABEL`=&lt;LEASE_LABEL&gt;  | EXPERIMENTAL |
```

Comment:
We've had problems with these types of metrics in the past, until we figure out a sustainable future for these types of metrics I would like to not introduce any new ones.

Reply:
Removed that metric.

Reply:
That makes sense but do you have a concrete use for it in an alerting rule or something? The renew time makes perfect sense and the owner makes sense to quickly find out when alerting which object the lease belongs so.

Reply:
I totally agree with you.